### DISCUSSION TITLE
Receive Syslogs from a Cisco switch

### DISCUSSION THREAD

**Discussion by loganterfehr** [Security-Onion-Solutions/securityonion#12152] [Unsupported Versions] [answered]
If I wanted to receive syslogs from a Cisco switch how would I go about doing it? Right now I have the logs being forwarded to my standalone manager node at 10.0.80.225. Would I need to open the firewall host groups for syslog up for the switch address? I am just a little confused and need some clarification since what I thought does not work. Let me know if you need more information. Thanks ahead of time all!

**Comment by reyesj2** [comment]
Yes, you need to allow syslog traffic from your cisco switch to the manager. https://docs.securityonion.net/en/2.4/firewall.html#configuration

https://docs.securityonion.net/en/2.4/syslog.html#syslog

**Reply by loganterfehr** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/122332360/abf84a7e-5688-4bd0-9651-a74a898e7345)

Just confirming, just need to allow the intended switches through the firewall using hostgroups whitelist? Assuming I have the Cisco switch correctly forwarding logs to my manager address I should be all set right and see logs coming from it in Kibana? I don't need to fidget with portgroups do I? Also thanks for the quick response.

**Reply by reyesj2** [reply, marked as answer]
Correct, you add to syslog hostgroup, you can also hit the button at the top under 'options' labled 'synchronize grid' to apply the changes to your grid. (Otherwise it takes about 15 minutes for the next highstate to run and changes to apply in the background)

Give it a few minutes, but if the switch is sending syslog data you should see it. You can use the SOC dashboard 'Syslog' to view syslog data 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/94730068/395802bb-b3d9-41bc-986b-9f48a71c3e7a)

**Reply by loganterfehr** [reply]
Okay yep, got them coming in now. Thanks for the help Jorge.
